lib/pluck_each.rb

```diff
@@ -16,10 +16,13 @@ def pluck_each(*args)
 
     def pluck_in_batches(*column_names)
       options = column_names.extract_options!
+      string_column_names = column_names.map(&:to_s)
```

Comment:
we do need to know if it had a single field, that field may be `first_name` ... which then the pluck call would have 2 fields even though the caller expects back just a single field